runtime/tools/spec-builder/test/json-files/customSpec-balances.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I think that a better name of this `json` file is `customSpec-noCode.json` rather than `-balances`